taxi/tests/test_models.py

```diff
@@ -0,0 +1,48 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+
+from taxi.models import Car, Manufacturer, Driver
+
+
+class ModelTests(TestCase):
+    def test_car_str(self):
+        manufacturer = Manufacturer.objects.create(
+            name="Test Manufacturer",
+            country="Test Country"
+        )
+        car = Car.objects.create(model="test", manufacturer=manufacturer)
+        self.assertEqual(str(car), car.model)
+
+    def test_driver_str(self):
+        driver = get_user_model().objects.create(
+            username="forest",
+            first_name="Fill",
+            last_name="Frost"
+        )
+        self.assertEqual(
+            str(driver),
+            f"{driver.username} ({driver.first_name} {driver.last_name})"
+        )

```

Comment:
Ensure that the `__str__` method of the `Driver` model returns the string in the format `{username} ({first_name} {last_name})`. If the actual implementation differs, adjust the expected string accordingly in the test.